packages/adblocker/src/resources.ts

```diff
@@ -25,9 +25,101 @@ export interface Resource {
   body: string;
 }
 
-// TODO - support # alias
+type ResourceBase = {
+  names: string[];
+  content: string;
+};
+
+export type RedirectResource = ResourceBase & {
```

Comment:
lets use old name `Resource` so it matches `resources` property of the `Resources` class

Reply:
Renamed to `Resource`, and also renamed `Resource` (which was originally used as an output type of `getResource`: `public getResource(name: string): Redirect & { dataUrl: string }`) to `Redirect`.